distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -282,6 +298,9 @@ public void setup() throws Exception {
 
     @After
     public void tearDown() throws Exception {
+
+        getCatalogBundle().setDownloadRetryDelayInSeconds(10);
```

Comment:
Since the retry delay is set to 1 in the `Before` method, this statement won't have an effect and can be removed. `Before` runs before each test and `After` runs after each test, so essentially this value will get set to 10 then immediately back to 1 between each test. The retry delay shouldn't have an impact on any itests, so it's fine to set it to 1 second and leave it.